docs/chapters/crud/index.md

```diff
@@ -0,0 +1,23 @@
+# CRUD Operations in Jina
+
+Paragraph. CRUD in general.
+
+Paragraph. Limitations until this point: only index and search. Incremental indexing.
+
+What are we doing now? Update, delete. 
+
+## Limitations
+
+Update at higher granularity?
```

Comment:
why is this a limitation?

Reply:
Cause it's not implemented yet.

Reply:
And I don't think it's part of the 1.0 timeline

Reply:
but `drivers` will work at any traversal paths, and should be out of the box?

Reply:
It's about this https://github.com/jina-ai/jina-hub/pull/1232#discussion_r538192452